modules/openapi-generator/src/main/resources/rust/reqwest/api.mustache

```diff
@@ -141,6 +141,27 @@ pub {{#supportAsync}}async {{/supportAsync}}fn {{{operationId}}}(configuration:
     {{/isApiKey}}
     {{/authMethods}}
     {{/hasAuthMethods}}
+    {{#hasAuthMethods}}
+    {{#withAWSV4Signature}}
+    if let Some(ref local_var_aws_v4_key) = local_var_configuration.aws_v4_key {
+        let local_var_new_headers = local_var_aws_v4_key.sign(
+	    &local_var_uri_str,
+	    "{{{httpMethod}}}",
+	    {{#hasBodyParam}}
+	    {{#bodyParams}}
+	    &serde_json::to_string(&{{{paramName}}}).unwrap(),
```

Comment:
While this is generally a [safe unwrap](https://docs.serde.rs/serde_json/fn.to_string.html) I'd think that an `expect` with some context would have been more useful.